### PR TITLE
Muon tidy up release notes and add image

### LINKED ISSUE #24918: Clean up Muon release notes
The muon release notes should be cleaned up before the next release.
This should include:
* Adding a new picture of the new GUI.

### PR DESCRIPTION
**Pull request by Matthew-Andrew**
**Description of work.**

This adds an image of the new GUI to the release notes

**Report to:** Anthony 

**To test:**

Check that the release notes display properly and read well also SPAG

Fixes #24918  #25080

*This does not require release notes* because it is release notes 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by Mantid-Matthew** [comment]
Marked as in progress until issue #25080 is also resolved.

**Comment by Mantid-Matthew** [comment]
@AnthonyLim23 I've added the images and deprecated warnings to this. I'm slightly uncertain what the major changes between the old and new versions of the Elemental Analysis GUI are could you update this bit.

File: docs/source/release/v4.0.0/muon.rst
Comment (AnthonyLim23) [inline docs/source/release/v4.0.0/muon.rst]: the old version might disappear after Friday. So we will edit this then

**Review by AnthonyLim23 — APPROVED** [review]

**Comment by martyngigg** [comment]
Should we just wait until Friday to merge this so we don't for get to update it?

**Comment by AnthonyLim23** [comment]
My concern is it will become a magnet for merge conflicts. I think the chances of removing it are high

**Comment by martyngigg** [comment]
Fair point. Can you make an issue to update that TODO when you get the information you need and mark it high priority for 4.0?

**Comment by martyngigg** [comment]
Actually we could just keep #25080 open. I think that will be fine.

**Pull request merged**
